### PR TITLE
Add Elements routing and simplify elements handling

### PR DESCRIPTION
**Pull request by Kapin**
Database wipe is needed from this. Last time we will need to do this.

Redid all the migrations because they broke horribly. Unused fields are hidden on elements.

### CODE REVIEW

File: src-backend/api/serializer.py [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+        fields = [field for field in self.fields.values() if not field.write_only]
+
+        for field in fields:
+            try:
+                attribute = field.get_attribute(instance)
+            except SkipField:
+                continue
+
+            if attribute is not None:
+                ret[field.field_name] = field.to_representation(attribute)
```
Comment (Trinovantes) [inline src-backend/api/serializer.py]: can this the moved into the try block?

**Comment by Trinovantes** [comment]
I don't really like stuffing everything into the `Element` class but if there's no other way, then it's fine. Can we instead use composition pattern?

**Comment by ConnorCimowsky** [comment]
LGTM

**Comment by Trinovantes** [comment]
+1

**Pull request merged**